terraform/modules/genericservice/main.tf

```diff
@@ -1,4 +1,8 @@
 
+locals {
+  storage_account_as_list = var.storage_account == null ? [] : [var.storage_account]
+}
+
```

Comment:
I think you should explain this, also the block below where you use it.

Reply:
sure, I thought so too after pushing, I will add comments.